tfc-dump.pl

```diff
@@ -109,16 +109,18 @@
 ### WARNING ###
 #
 # This code assumes that all of the TFC Variable Sets are contained within
-# the first result page of 20 entries.  This was true for SIL in December 2022.
+# the first result page of 100 entries.  This was true for SIL in August 2024.
 #
 ####
 
 my @vs_names;
 my @vs_ids;
+my $pg_size = 100;
+my $total_count;
 my $tmpfile = `mktemp`;
 chomp($tmpfile);
 
-$curl_query = "\"https://app.terraform.io/api/v2/organizations/${tfc_org_name}/varsets\"";
+$curl_query = "\"https://app.terraform.io/api/v2/organizations/${tfc_org_name}/varsets?page%5Bsize%5D=${pg_size}\"";
```

Comment:
Had to look up page%5Bsize%5D -- it's the square braces.

Reply:
Correct.  The documentation said I might need to do it that way so I did just to avoid any problems.  I tried using the square brackets this morning and confirmed they didn't work (`curl: (3) bad range in URL...`).